docs/structure.rst

```diff
@@ -1,127 +1,128 @@
 Overview of directories and files
 =================================
 
--  ``docs/``: documentation in `ReStructured Text`_ format, to be built with
+*  ``docs/``: documentation in `ReStructured Text`_ format, to be built with
    `Sphinx`_.
 
-   - ``docs/README.md``: introduction to the user manual and guidelines
+   * ``docs/README.md``: introduction to the user manual and guidelines
 
--  ``editors/``: editor plugins for handling Lambdapi files
+*  ``editors/``: editor plugins for handling Lambdapi files
 
-   -  ``emacs/``: code for emacs
-   -  ``vim/``: code for vim
-   -  ``vscode/``: code for vscode
+   *  ``emacs/``: code for emacs
+   *  ``vim/``: code for vim
+   *  ``vscode/``: code for vscode
 
-      -  ``.vscode/*.json``: config for launching, debugging, building
+      *  ``.vscode/*.json``: config for launching, debugging, building
          the extension
-      -  ``src/*.ts``: the VSCode plugin source code written in the
+      *  ``src/*.ts``: the VSCode plugin source code written in the
          TypeScript language
-      -  ``package.json``: the manifest of the plugin (activation
+      *  ``package.json``: the manifest of the plugin (activation
          events, scripts, dependencies, …)
-      -  ``tsconfig.json``: TypeScript configuration (directories, …)
-      -  ``vscode.proposed.d.ts``: VSCode API (Microsoft file, likely to
+      *  ``tsconfig.json``: TypeScript configuration (directories, …)
+      *  ``vscode.proposed.d.ts``: VSCode API (Microsoft file, likely to
          change)
-      -  ``lp.configuration.json``: specific characters
-      -  ``syntaxes/lp.tmLanguage.json``: grammar of lambdapi
-      -  ``Makefile``
+      *  ``lp.configuration.json``: specific characters
+      *  ``syntaxes/lp.tmLanguage.json``: grammar of lambdapi
+      *  ``Makefile``
 
--  ``libraries/``: libraries of Dedukti files (see GNUmakefile)
+*  ``libraries/``: libraries of Dedukti files (see GNUmakefile)
 
--  ``src/core/``: source code of Lambdapi
+* ``src/lplib/``: extensions of the Ocaml standard library for Lambdapi
 
-   -  utilities:
+  *  ``stubs.c``: C implementation for ``Extra.Filename.realpath``.
```

Comment:
lplib/range* files are missing. Should they be in lplib or backbone?

Reply:
Ah good question, I'm not sure.